src/environments/environment.ts

```diff
@@ -22,6 +22,9 @@ export const GROUPS = {
   ADMIN: 'time-tracker-admin',
   TESTER: 'time-tracker-tester',
 };
+export const TOGGLES = {
+  SWITCH_GROUP: 'switch-group',
```

Comment:
I have a question:
The functionality of this var is the storage of the state of the switch group?
- if this value changes this value is updated?

Reply:
No, it just the name of FT. I was using because it was easier to change the name if it necesary and in case you needed to use it in more parts of the application. The value isn't update if FT changes